Tech writer review edits (#17229)
newrelic/docs-website: src/content/docs/vulnerability-management/change-vulnerability-status.mdx

import vmIgnoreBulkDetails from 'images/security_screenshot-ignore-bulk-details.webp'

import vmIgnoreBulkForm from 'images/security_screenshot-ignore-bulk-form.webp'

import vmAffectedEntityVulnList from 'images/security_screenshot-affected-entity-vulnlist.webp'

import vmAffectedEntityDetails from 'images/security_screenshot-affected-entity-details.webp'

import vmAffectedEntityFiltered from 'images/security_screenshot-affected-entity-filtered.webp'

import vmAffectedEntityForm from 'images/security_screenshot-affected-entity-form.webp'

import vmAffectedBulkVulnList from 'images/security_screenshot-affected-bulk-vulnlist.webp'

import vmAffectedBulkSelection from 'images/security_screenshot-affected-bulk-selection.webp'

import vmAffectedBulkDetails from 'images/security_screenshot-affected-bulk-details.webp'

import vmAffectedBulkForm from 'images/security_screenshot-affected-bulk-form.webp'

New Relic Vulnerability Management may detect an overwhelming amount of vulnerabilities across your system. Most of these vulnerabilities may need remediation, but many might be low risk, or have limited external exposure. You can manage your vulnerabilities by:

> * Assigning individual entities the Ignored status so their vulnerabilities no longer appear in Vulnerability Management

Reviewer: ```suggestion
* Assigning individual entities the `Ignored` status so their vulnerabilities no longer appear in Vulnerability Management
```

* Ignoring a certain kind of vulnerability in bulk so it no longer appears across multiple entities 

> * Removing the Ignored status at a later date if you want to detect those vulnerabilities again

Reviewer: ```suggestion
* Removing the `Ignored` status at a later date if you want to detect those vulnerabilities again
```

## Ignore a vulnerability [#ignore-vulnerability]

> You can manually ignore vulnerabilities on a per entity basis. Once you set the status of an instance to Ignored, we’ll exclude this instance from the summary tiles and vulnerability list by default. Other instances of this vulnerability on other entities will still keep the Affected status.

Reviewer: ```suggestion
You can manually ignore vulnerabilities on a per entity basis. Once you set the status of an instance to `Ignored`, we’ll exclude this instance from the summary tiles and vulnerability list by default. Other instances of this vulnerability on other entities will still keep the `Affected` status.
```

<CollapserGroup>
    <Collapser
        id="ignore-entity"
        title="Ignore a vulnerability for a specific entity"
    >

> 1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select your entity) > **Vulnerability Management** > Click the **Vulnerabilities** tab, then select the vulnerability you want to ignore. If you don't see the vulnerability you want to manage, it might already have the Ignored status. 

Reviewer: ```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select your entity) > **Vulnerability Management** > Click the **Vulnerabilities** tab, then select the vulnerability you want to ignore. If you don't see the vulnerability you want to manage, it might already have the `Ignored` status. 
```

    <img
    title="Vulnerabilities tab scoped to a specific entity"
    alt="An image showing the scoped vulnerabilities tab of Vulnerability Management"
    src={vmIgnoreEntityVulnList}
    />

> 1. To update the status to Ignored, click the **Change to ignored** button. 

Reviewer: ```suggestion
1. To update the status to `Ignored`, click the **Change to ignored** button. 
```

        <img
            title="Vulnerability instance details pages"
            alt="An image showing the change to ignored button on vulnerability instance details page"
            src={vmIgnoreEntityDetails}
        />

> 1. Fill out the form to explain the reasoning behind changing this vulnerability to Ignored. Include a time period that the vulnerability should be ignored.

Reviewer: ```suggestion
1. Fill out the form to explain the reasoning behind ignoring this vulnerability. Include a time period that the vulnerability should be ignored.
```

    <img
        title="Vulnerability instance details form"
        alt="An image showing the ignore form on vulnerability instance details page"
        src={vmIgnoreEntityForm}
    />

    </Collapser>
    <Collapser
        id="bulk-ignore"
        title="Ignore a vulnerability across multiple instances"
    >

> 1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > Click the **Vulnerabilities** tab 

Reviewer: ```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > click the **Vulnerabilities** tab 
```

> 1. Click the vulnerability you want to ignore. By default, this vulnerability view is filtered to only show vulnerabilities where at least one entity is marked Affected.

Reviewer: ```suggestion
1. Click the vulnerability you want to ignore. By default, this vulnerability view is filtered to only show vulnerabilities where at least one entity is marked `Affected`.
```

    <img
        title="Vulnerabilities tab for all entities"
        alt="An image showing the all entities vulnerabilities tab of Vulnerability Management"
        src={vmIgnoreBulkVulnList}
    />

1. Referring to **Area of Impact**, select each entity you want to ignore, then click the **Change to ignored** button.
    
    <img
        title="Vulnerabilities details"
        alt="An image showing the selection of specific entities"
        src={vmIgnoreBulkDetails}
    />

> 1. Fill out the form to explain the reasoning behind changing this vulnerability to Ignored. Include a time period that the vulnerability should be ignored.

Reviewer: ```suggestion
1. Fill out the form to explain the reasoning behind changing this vulnerability to `Ignored`. Include a time period that the vulnerability should be ignored.
```

Author: Standardized this to say "Fill out the form to explain the reasoning behind ignoring th is vulnerability."

        <img
            title="Vulnerabilities details form "
            alt="An image showing the ignore form on vulnerability instance details page"
            src={vmIgnoreBulkForm}
        />
    </Collapser>
</CollapserGroup>

## Change Ignored status to Affected [#bulk-manage]

> It's good practice to review ignored vulnerabilities and validate whether they should stay ignored. Changing the status from Ignored to Affected reintroduces the vulnerability back into default views andcounts on this entity, but other instances of this vulnerability on other entities will still remain ignored.

Reviewer: ```suggestion
It's good practice to review ignored vulnerabilities and validate whether they should stay ignored. Changing the status from Ignored to Affected reintroduces the vulnerability back into default views and counts on this entity, but other instances of this vulnerability on other entities will still remain ignored.
```

Author: changed this locally!

<CollapserGroup>
    <Collapser
        id="remove-ignored"
        title="Remove ignored status for a specific entity"
    >

> 1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select the entity you want to update) > **Vulnerability Management** > Click the **Vulnerabilities** tab.

Reviewer: ```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select the entity you want to update) > **Vulnerability Management** > click the **Vulnerabilities** tab.
```

1. View all your ignored vulnerabilities by updating the filter from `Status = Affected` to `Status = Ignored`. 

    <img
    title="Vulnerabilities tab scoped to a specific entity"
    alt="An image showing the scoped vulnerabilities tab of Vulnerability Management with filter tab open"
    src={vmAffectedEntityVulnList}
    />

1. Choose the vulnerability that you want to update, then click **Change to affected**.

    <img
        title="Vulnerabilities tab scoped to a specific entity and only ignored instances"
        alt="An image showing the scoped vulnerabilities tab of Vulnerability Management filtered to only ignored instances"

>         src={vmAffectedEntityFiltered}

Reviewer 2: Maybe this should be the other visual vmAffectedEntityDetails
The image you are using here is still on the vulnerability list with it filtered down but not the one with the 'Change to affected' button?

Author: Updated!

    />

> 1. Fill out the form to explain the reasoning behind changing this vulnerability to Affected.

Reviewer: ```suggestion
1. Fill out the form to explain the reasoning behind changing this vulnerability to `Affected`.
```

        
    <img
        title="Vulnerability instance details form"
        alt="An image showing the affected form on vulnerability instance details page"
        src={vmAffectedEntityForm}
    />

    </Collapser>
    <Collapser
        id="bulk-change-status"
        title="Remove Ignored status across multiple instances"
    >

> 1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > Click the **Vulnerabilities** tab 

Reviewer: ```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > click the **Vulnerabilities** tab 
```

> 1. From the settings cog, select the **Ignored** checkbox. This adds an ignore column to your vulnerability table.

Reviewer: ```suggestion
1. From the settings cog, click the **Ignored** checkbox. This adds an ignore column to your vulnerability table.
```

1. Select a vulnerability with an ignored count greater than 0. 

    <img
        title="Vulnerabilities tab with ignored column added"
        alt="An image showing the all entities vulnerabilities tab with an ignore column added"
        src={vmAffectedBulkVulnList}
    />

> You can also bulk manage your Ignored vulnerabilities with the filter bar. This method is better if you X:

Reviewer 2: This sentence and restarting of the steps really doesn't make sense to me?

Author: ah ok! that makes sense. i'll merge 'em in then

> You can also bulk manage your Ignored vulnerabilities with the filter bar. This method is better if you X:

Reviewer: ```suggestion
You can also bulk manage your ignored vulnerabilities with the filter bar. This method is better if you have a lot of similar vulnerabilities to update at once:
```

1. Update the filter from `Status = Affected` to `Status = Ignored`. 

    <img
        title="Vulnerabilities details"
        alt="An image showing the vulnerability details page with the status dropdown open"
        src={vmAffectedBulkDetails}
    />

> 1. Referring to the **Area of Impact** section, select the entities you want to mark Affected.

Reviewer: ```suggestion
1. In the **Area of Impact** section, select the entities you want to mark `Affected`.
```

    <img
        title="Vulnerabilities details with instances selected"
        alt="An image showing the vulnerability details page with affected instances selected"
        src={vmAffectedBulkSelection}
    />

> 1. Fill out the form to explain the reasoning behind changing this vulnerability to Affected.

Reviewer: ```suggestion
1. Fill out the form to explain the reasoning behind changing this vulnerability to `Affected`.
```

    <img
        title="Vulnerabilities details with form open"
        alt="An image showing the vulnerability details page with form to change instances to affected"
        src={vmAffectedBulkForm}
    />

    </Collapser>
</CollapserGroup>

> ## Surface Ignored vulnerabilities [#surface-vulnerabilities]

Reviewer 2: We should also include back the 'settings icon' as a way to unhide the ignored column on the vulnerability list page.

Author: Updated!

> Before you assign **Ignored** to a vulnerability or entity, it's important to understand that the change affects how other code owners manage that particular instance. For example, marking a vulnerability or entity as **Ignored** prevents other users with account access from seeing that vulnerability. 

Reviewer: Let's get consistent: Should we put things like Ignored and Affected in ** or code backticks? Whatever we do, we should it every time

* You can surface **Ignored** vulnerabilities with the filter bar. From the filter bar, you can add `Status = Ignored` to view Ignored vulnerabilities. 
* As best practice, we recommend reviewing your vulnerabilities at minimum once every 90 days to ensure you aren't introducing added risk into your system.

## Why ignore a vulnerability? [#why-ignore]

> When you choose to ignore a vulnerability, you'll be prompted to explain the rationale behind the decision. For example, maybe your system uses some package that contains a mixture of deprecated and supported code. Your system may depend on the supported code but not the deprecated code. You can choose to ignore the vulnerability notice as it's irrelevant to your system. This is good context to provide in the form, and we recommend you provide as much documentation to corroborate it as possible. 

Reviewer 2: I don't think this phrased correctly. 
Deprecated code and supported code is different than 'not using the vulnerable function'. 

I think this section should be reworded again to target the 'vulnerable method is not used'.
Happy to hop on a call to chat about this again.

Author: sent you example/updated copy via slack!

> When ignoring a dependency, you can set a duration for the Ignored s tatus. When the time period passes, Vulnerability Management will perform an automatic update that changes it from Ignored to affected. If an instance is already considered **No Longer Detected**, the status will remain no longer detected.

Reviewer: ```suggestion
When ignoring a dependency, you can set a duration for the Ignored status. When the time period passes, Vulnerability Management will perform an automatic update that changes it from Ignored to affected. If an instance is already considered **No Longer Detected**, the status will remain no longer detected.
```
